src/Extension/FluentExtension.php

```diff
@@ -161,6 +151,52 @@ public function getLocalisedFields($class = null)
     }
 
     /**
+     * @param string $field
+     * @param string $type
+     * @param string|null $class
+     * @return bool
+     */
+    public function isFieldLocalised($field, $type, $class = null)
+    {
+        if (!$class) {
+            $class = get_class($this->owner);
+        }
+
+        // Data and field filters
+        $fieldsInclude = Config::inst()->get($class, 'field_include');
+        $fieldsExclude = Config::inst()->get($class, 'field_exclude');
+
+        if ($fieldsInclude && in_array($field, $fieldsInclude)) {
+            return true;
+        }
+
+        if ($fieldsExclude && in_array($field, $fieldsExclude)) {
```

Comment:
It'll be simpler to stick with the anyMatch() method rather than mixing comparisons. Remove these two in_array checks.